_posts/this-week-in-solid/next.md

```diff
@@ -77,6 +77,7 @@ If you have any new apps, Pods, or identity providers, or there are updates to e
 * [Darcy Ibex](https://ibex.darcy.is) got made in a hackathon. Darcy Ibex is a prototype social media stream based entirely on Solid
 * [Pod Homepage](https://gitlab.com/angelo-v/pod-homepage), a potential homepage for your Solid Pod ([Live Example](https://angelo.veltens.org/)).
 * [Solidarity](https://scenaristeur.github.io/solidarity/) is the start of a new chat forum project based on LongChat and inbox
+* [Golf Course Assistant](https://gca-solid.now.sh/) is an app where golfers can track their score on the course. Build with the solid-react SDK
```

Comment:
Small typo:

```suggestion
* [Golf Course Assistant](https://gca-solid.now.sh/) is an app where golfers can track their score on the course. Built with the solid-react SDK
```

Reply:
Thanx!

Reply:
@michaud You have to click "Commit suggestion" to actually incorporate the fix - I don't have the rights to change your PR for you :)